consensus/exit_probability_validator/exit_probability_validator.c

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include "consensus/exit_probability_validator/exit_probability_validator.h"
+#include "utils/logger_helper.h"
+
+#define WALKER_VALIDATOR_LOGGER_ID "consensus_walker_validator"
+
+retcode_t iota_consensus_exit_prob_transaction_validator_init(
+    const tangle_t *tangle, const milestone_t *milestone,
+    const ledger_validator_t *lv, exit_prob_transaction_validator_t *wv) {
+  logger_helper_init(WALKER_VALIDATOR_LOGGER_ID, LOGGER_INFO, true);
+  wv->tangle = tangle;
+  wv->milestone = milestone;
+  wv->lv = lv;
+  return RC_OK;
+}
+
+retcode_t iota_consensus_exit_prob_transaction_validator_destroy(
+    exit_prob_transaction_validator_t *wv) {
+  logger_helper_destroy(WALKER_VALIDATOR_LOGGER_ID);
+  wv->tangle = NULL;
+  wv->milestone = NULL;
+  wv->lv = NULL;
+}
+retcode_t iota_consensus_exit_prob_transaction_validator_is_valid(
+    const exit_prob_transaction_validator_t *ep_validator, trit_array_p tx_hash,
+    bool *is_valid) {
+  retcode_t ret = RC_OK;
+
+  iota_stor_pack_t tx_pack;
```

Comment:
You can initialize it in one line like
`iota_stor_pack_t tx_pack = {(void **)(&tx_models), 1, 0, false};`